echopype/calibrate/cal_func.py

```diff
@@ -0,0 +1,24 @@
+from .calibrate_ek import CalibrateEK60, CalibrateEK80
+from .calibrate_azfp import CalibrateAZFP
+
+CALIBRATOR = {
+    'EK60': CalibrateEK60,
+    'EK80': CalibrateEK80,
+    'AZFP': CalibrateAZFP
+}
+
+
+def get_Sv(echodata, env_params=None, cal_params=None):
```

Comment:
I was thinking maybe having a generalized function for the calibration similar to like `to_file`. I think this would make things clearer API wise. I can see it being like:

```python
from echopype import EchoData
from echopype import calibrate

echodata = EchoData(...)

Sv = calibrate(echodata, cal_type='Sv')
```

Something like that.

Reply:
That is actually how it is structured now (see #250). There are other functions under calibrate that are forthcoming, which is why I have it as
From echopype import calibrate
Sv = calibrate.get_Sv(echodata)

Reply:
Some of the other functions would require passing in other arguments, so it'll be too messy to use the same cal_type set up.

Reply:
Ah okay that makes sense. Thanks for clearing that up :smile:

Reply:
Thinking about it again, with the current structure, for different functions (let's say there are `calibrate.get_Sv(echodata)` and `calibrate.get_single_target_TS(echodata, target_location)`) under the hood each call creates and destroys a Calibrate object. Can this potentially be a problem?

Reply:
I think it would be better if we can do the creation of Calibrate object once. Can the echodata object contain the calibrate object internally? 

You're creating this with echodata attributes already anyways:

```python
cal_obj = CALIBRATOR[echodata.sonar_model](echodata)
```

Reply:
Heh, that was actually how I had it before, and you advised against having it as part of echodata! 😛

Regardless, yes massaging calibrate.get_Sv to use internal calibrator object in echodata is straightforward to do.

Reply:
> Heh, that was actually how I had it before, and you advised against having it as part of echodata! 

I advised against having the actual `get_Sv` function as part of echodata :stuck_out_tongue: hehe. Anyways, now thinking about it further I think the `cal_obj` can probably live in echodata.